tests/python_tests/common.py

```diff
@@ -71,59 +71,105 @@ def run_llm_pipeline(
-    scheduler_config = dict_to_scheduler_config(scheduler_params)
+# def run_cb_pipeline_with_ref(tmp_path: str,
+#                              model_id: str,
+#                              scheduler_params: dict = {},
+#                              generation_config : GenerationConfig | dict = None):
+#     prompts, generation_configs = get_test_dataset()
+#     scheduler_config = dict_to_scheduler_config(scheduler_params)
 
-    # override dataset's generation config
-    if generation_config is not None:
-        if type(generation_config) is dict:
-            generation_config = GenerationConfig(**generation_config)
-        generation_configs = [generation_config] * len(prompts)
+#     # override dataset's generation config
+#     if generation_config is not None:
+#         if type(generation_config) is dict:
+#             generation_config = GenerationConfig(**generation_config)
+#         generation_configs = [generation_config] * len(prompts)
 
-    opt_model, hf_tokenizer, models_path = download_and_convert_model(model_id, tmp_path)
+#     opt_model, hf_tokenizer, models_path = download_and_convert_model(model_id, tmp_path)
 
-    hf_results = run_hugging_face(opt_model, hf_tokenizer, prompts, generation_configs)
-    ov_results = run_continuous_batching(models_path, scheduler_config, prompts, generation_configs)
+#     hf_results = run_hugging_face(opt_model, hf_tokenizer, prompts, generation_configs)
+#     ov_results = run_continuous_batching(models_path, scheduler_config, prompts, generation_configs)
 
-    compare_generation_results(prompts, hf_results, ov_results, generation_configs)
+#     compare_generation_results(prompts, hf_results, ov_results, generation_configs)
 
 
 # TODO: remove after Generator property is supported by LLMPipeline / VLMPipeline
-def generate_and_compare_with_reference_text(models_path: Path,
-                                             prompts: List[str],
-                                             reference_texts_per_prompt: List[List[str]],
-                                             generation_configs: List[GenerationConfig],
-                                             scheduler_config: SchedulerConfig):
-    ov_results : List[GenerationResult] = run_continuous_batching(models_path, scheduler_config, prompts, generation_configs)
-
-    assert len(prompts) == len(reference_texts_per_prompt)
-    assert len(prompts) == len(ov_results)
-
-    for prompt, ref_texts_for_this_prompt, ov_result in zip(prompts, reference_texts_per_prompt, ov_results):
-        print(f"Prompt = {prompt}\nref text = {ref_texts_for_this_prompt}\nOV result = {ov_result.m_generation_ids}")
-
-        assert len(ref_texts_for_this_prompt) == len(ov_result.m_generation_ids)
-        for ref_text, ov_text in zip(ref_texts_for_this_prompt, ov_result.m_generation_ids):
-            assert ref_text == ov_text
-
+# def generate_and_compare_with_reference_text(models_path: Path,
```

Comment:
should we remove commented code completely?

Reply:
of course, yes. I forget to add labels `do_not_review`, `do_not_merge`.